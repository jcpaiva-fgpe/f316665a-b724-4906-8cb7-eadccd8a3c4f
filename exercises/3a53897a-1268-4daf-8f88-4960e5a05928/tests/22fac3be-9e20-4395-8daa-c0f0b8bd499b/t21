12778 20556 12780 20558
5
4 4
-20 -20
20 20
-4 -4
-2 2
