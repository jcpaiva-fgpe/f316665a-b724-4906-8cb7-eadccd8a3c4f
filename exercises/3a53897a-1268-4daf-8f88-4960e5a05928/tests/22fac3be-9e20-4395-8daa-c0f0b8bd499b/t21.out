1
Atenas 5
